docs/conformance.md

```diff
@@ -14,8 +14,10 @@ anything outside of the explicitly created test namespaces by the binary.
 To execute the conformance tests for a released conformance test suite, run:
 
 ```bash
-VERSION=v0.33.0
-sonobuoy run --plugin https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/conformance.yaml
+VERSION=v0.41.0
```

Comment:
maybe rename VERSION to CONFORMANCE_VERSION

and https://github.com/kubevirt/kubevirt/pull/6199/files#diff-013a891077210c9a4659c9288e6f1e60cefda476edef377793981451a3d75b48R69
this one, rename to CONFORMANCE_LATEST ?

it will be more self explanatory i think

Reply:
> maybe rename VERSION to CONFORMANCE_VERSION

I think its better to keep as is, `VERSION` refer to Kubevirt release version not specifically for conformance

> and https://github.com/kubevirt/kubevirt/pull/6199/files#diff-013a891077210c9a4659c9288e6f1e60cefda476edef377793981451a3d75b48R69
> this one, rename to CONFORMANCE_LATEST ?
> 
> it will be more self explanatory i think

This one also refers to Kubevirt latest nightly build

Reply:
ok than to KUBEVIRT_VERSION and KUBEVIRT_LATEST ?

Reply:
Done